From 57666c845bf57c4fca1a04f243df8bf47ec9d09a Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Fri, 23 Mar 2018 17:34:55 -0500
Subject: [PATCH 3541/5242] MLK-17703-8: drm: imx: dcss: default output pipe
 gamut to REC709

commit  b047e14e647948e3e2ddd6b7f0d94a2793cd35a6 from
https://source.codeaurora.org/external/imx/linux-imx.git

In case we don't get any information about colorimetry from HDMI sink,
set default gamut to REC709.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/dcss/dcss-crtc.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/dcss/dcss-crtc.c b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
index b7eeade..5e80bc2 100644
--- a/drivers/gpu/drm/imx/dcss/dcss-crtc.c
+++ b/drivers/gpu/drm/imx/dcss/dcss-crtc.c
@@ -176,10 +176,8 @@ void dcss_crtc_setup_opipe(struct drm_crtc *crtc, struct drm_connector *conn,
 		dcss_crtc->opipe_g = G_REC2020;
 	else if (colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_ADOBE_RGB))
 		dcss_crtc->opipe_g = G_ADOBE_ARGB;
-	else if (colorimetry & BIT(HDMI_EXTENDED_COLORIMETRY_XV_YCC_709))
-		dcss_crtc->opipe_g = G_REC709;
 	else
-		dcss_crtc->opipe_g = G_REC601_PAL;
+		dcss_crtc->opipe_g = G_REC709;
 
 	if (eotf & (1 << 2))
 		dcss_crtc->opipe_nl = NL_REC2084;
-- 
1.7.9.5

